modules/doc/content/newsletter/2023/2023_05.md

```diff
@@ -22,20 +17,90 @@ Several `Positions` were implemented to help kickstart the use of the system:
 - [ElementCentroidPositions.md]
 - [ElementGroupCentroidPositions.md]
 
+Positions can now also be used to match source and target locations during transfers.
+By specifying the [!param](/Transfers/MultiAppGeneralFieldNearestNodeTransfer/use_nearest_position),
+target locations in the target application will only be matched with source locations that
+are nearest to the same `Position` than any of the other `Position` in the `Positions` object.
+
 ### Possibility to combine TimeSteppers
 
 The time stepper system in MOOSE has been improved. A new [`[TimeSteppers]`](TimeSteppers/index.md) block has been introduced to replace the functionality of the current `[TimeStepper]` block. The new system keeps the features of `[TimeStepper]` and also supports composing multiple time steppers to compute the time step size. Users can still use `[TimeStepper]` for now, but in the future this block will be deprecated.
 
 When more than one time stepper is provided, the new time stepper system will automatically compute the minimum time step among all the input time steppers as the time step size to use for that application. The new time stepper system also provides a [!param](/Executioner/TimeSteppers/lower_bound) parameter to further control the time stepper size (e.g. enable growth driven by a single time stepper). This system is controllable via the [Controls](syntax/Controls/index.md) system. The user can turn on/off the provided time steppers to control the usage of time steppers ; for example, one could make time stepper(s) only active [during a certain time period](TimePeriod.md).
 
-## libMesh-level Changes
+## Modules improvements
+
+### Fluid properties for the NaK eutectic
+
+Fluid properties were added for the `NaK` eutectic in the fluid properties module.
+These are extracted from the public Sodium handbook chapter on fluid properties \cite().
+The fluid properties implemented should suffice for use with the Navier Stokes module.
 
-## PETSc-level Changes
+### Functor solid thermal properties
+
+A functor version of the solid thermal properties material has been added to the solid properties module.
+Functor materials are more flexible, enabling all types of arbitrary evaluations on faces or points with an element for example.
+This addition is leveraged by the finite volume heat conduction solver.
 
 ## Bug Fixes and Minor Enhancements
 
+- Moving source points in Dirac Kernels, such as the [ReporterPointSource.md] are now supported.
+- A [ParsedVectorAux.md] has been added to MOOSE. It enables forming in a very
+  flexible manner vector auxiliary variables. It can be used for initialization
```

Comment:
```suggestion
- A [ParsedVectorAux.md] has been added to MOOSE. It enables forming vector
  auxiliary variables in a very flexible manner, and it can be used for initialization
```